frameworks/PHP/swoole/benchmark_config.json

```diff
@@ -2,68 +2,93 @@
   "framework": "swoole",
   "tests": [{
     "default": {
+      "dockerfile": "swoole.dockerfile",
       "json_url": "/json",
       "plaintext_url": "/plaintext",
       "db_url": "/db",
-      "query_url": "/db?queries=",
+      "query_url": "/query?queries=",
       "fortune_url": "/fortunes",
       "update_url": "/updates?queries=",
       "port": 8080,
       "approach": "Realistic",
       "classification": "Platform",
-      "database": "MySQL",
+      "database": "postgres",
       "framework": "swoole",
       "language": "PHP",
-      "flavor": "PHP8.1",
+      "flavor": "PHP8.2",
       "orm": "Raw",
       "platform": "swoole",
       "webserver": "none",
       "os": "Linux",
       "database_os": "Linux",
-      "display_name": "Swoole",
+      "display_name": "swoole",
       "notes": "",
       "versus": "php"
     },
-    "postgres": {
```

Comment:
If you change the name, we will loose all the history in all the rounds.
Try to maintain the name, and only change the display name.
Of course, try to use the same database.